index.html

```diff
@@ -96,13 +97,214 @@
                 <i class="fa-solid fa-arrow-right"> </i>
             </button>
 
-            <div class="slider__indicators d-flex justify-content-center pt-5 pb-2 pt-xs-4 pb-xs-2 gap-4">
+            <div class="slider__indicators d-flex justify-content-center pt-5 pb-2 pt-xs-5 pb-xs-2 gap-4">
                 <div class="slider__indicator slider__indicator--current"></div>
                 <div class="slider__indicator"></div>
                 <div class="slider__indicator"></div>
             </div>
         </div>
     </div>
 
+    <!-- Content -->
+    <div class="content">
+        <div class="content__container container d-flex flex-column align-items-center pt-11 pt-xs-11 pb-2">
+            <div class="col-lg-12 px-0 pb-11 d-flex flex-wrap justify-content-center">
+                <!-- Introduction -->
+                <div class="introduction d-flex flex-column align-items-center pt-3 pt-xs-3 ">
+                    <h1 class="introduction__title m-0">Lorem <span>ipsum</span></h1>
+                    <p class="introduction__text m-0 pt-8 pt-xs-8 pb-7 pb-xs-7">
+                        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+                        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+                        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+                        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                         Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, 
+                         totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. 
+                         Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, 
+                         sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. 
+                         Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, 
+                         adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. 
+                        Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit
+                    </p>
+                </div>
+
+                <!-- Pricing Table 1 -->
+                <div class="col-lg-4 col-md-6 col-sm-12 d-flex flex-column pricing-table pt-11 px-0 ">
```

Comment:
Ở màn hình này pricing table to quá. Em cho cách sang 2 bên nhiều hơn nhá
![image](https://github.com/user-attachments/assets/5d30948a-d220-4bd9-b19d-9c378084cf06)